ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -103,8 +103,7 @@ message CreatedEvent {
   google.protobuf.StringValue agreement_text = 6;
 
   // Metadata of the contract. Required for contracts created
-  // after the introduction of explicit disclosure. If explicit disclosure is disabled,
-  // this field will be empty.
+  // after the introduction of explicit disclosure.
```

Comment:
@mziolekda I have adapted this description to be truthful, since this change always populates the contract metadata in LAPI streams with the `create_key_hash` and `created_at`. Doing so since it is less impactful in code and has a very low payload overhead compared to the current state. Please let me know if you disagree

Reply:
:+1: